lib/client/profile_e2e_test.go

```diff
@@ -48,14 +48,14 @@ import (
 var profileSpec1 = api.ProfileSpec{
 	IngressRules: []api.Rule{testutils.InRule1, testutils.InRule2},
 	EgressRules:  []api.Rule{testutils.EgressRule1, testutils.EgressRule2},
-	Tags:         []string{"profile1-tag1", "profile1-tag2"},
 }
+var tags1 = []string{"profile1-tag1", "profile1-tag2"}
```

Comment:
Any reason we can't inline these in the tests below rather than creating these special variables?